client/src/App.js

```diff
@@ -16,24 +14,32 @@ function App() {
   return (
     <div className="App">
       <Router>
-        <Home />
-        <Box
-          sx={{
-            width: "100%",
-            height: "100vh",
-            backgroundImage: "linear-gradient(to bottom right, #720E7A, #0F1A37)",
-          }}
-        >
-          <DummyWordHolder letterState={letterState} setLetterState={setLetterState} />
-          <HolderForWrongLetters letterState={letterState} />
-          <DummyInputHolder />
-
-        </Box>
+        <Routes>
+          <Route
+            path="/start-new-game"
+            element={
+              <Box
+                sx={{
+                  width: "100%",
+                  height: "100vh",
+                  backgroundImage:
+                    "linear-gradient(to bottom right, #720E7A, #0F1A37)",
+                }}
+              >
+                <DummyWordHolder
```

Comment:
i have a questions about this one: why we have openning page as start new game? should it be a separate route?